python/wpt/export.py

```diff
@@ -21,21 +21,24 @@
 
 
 def main() -> int:
-    context = json.loads(os.environ['GITHUB_CONTEXT'])
+    #context = json.loads(os.environ['GITHUB_CONTEXT'])
```

Comment:
I guess changes in this file were for testing?

Reply:
Yikes, yeah.